docs/tutorials.rst

```diff
@@ -194,26 +200,28 @@ example.
 
    import random
    import string
-
+   
```

Comment:
PEP8 recommend surround top-level function and class definitions with two blank lines. According to reStructuredText syntax code block must be indented. Trailing whitespaces are the remains after the indentation using Column Mode Editing (Rectangular Selection).